staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -819,7 +833,9 @@ func (c completedConfig) New(name string, delegationTarget DelegationTarget) (*G
 		APIServerID:           c.APIServerID,
 		StorageVersionManager: c.StorageVersionManager,
 
-		Version: c.Version,
+		EffectiveVersion: c.EffectiveVersion,
+		Version:          c.Version,
```

Comment:
does David comment about `c.Version` applies here too https://github.com/kubernetes/kubernetes/pull/122891/commits/c3f12154cf22bc473e7e3b6b8d1d776e41b9a585#r1628463485 ?

Reply:
It is removed in the latest commits. Do you know they should not be removed?